Modules/_sre/sre.h

```diff
@@ -73,9 +73,9 @@ typedef struct {
     Py_ssize_t pos, endpos;
     int isbytes;
     int charsize; /* character size */
-    /* registers */
-    Py_ssize_t lastindex;
-    Py_ssize_t lastmark;
+    /* marks */
```

Comment:
I think it is worth to move `match_all` and `must_advance` above this comment. They are not related to marks.

Reply:
done